src/Components/FiveDayForecastCard/FiveDayForecastCard.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {FC} from "react";
+import { Forecast, DailyForecast } from "../../model/Weather";
+import './FiveDayForecastCard.css'
+
+interface FiveDayForecastCardProps {
+  key: number;
+  temp: number;
+  days: Forecast | null;
+}
+
+export const FiveDayForecastCard: FC<FiveDayForecastCardProps> = ({ key, temp, days }) => {
+  if (days) {
+    return (
+      <>
```

Comment:
I wonder if we could map over this and clean it up by making another component for DayForecastDetails?